src/components/MyAccount/ProfileHeader.tsx

```diff
@@ -53,19 +74,32 @@ const ProfileHeader = ({ patron }: { patron: Patron }) => {
     .map(buildListElementsWithIcons)
 
   return (
-    <List
-      className={styles.myAccountList}
-      id="my-account-profile-header"
-      title="My Account"
-      type="dl"
-      sx={{
-        border: "none",
-        h2: { border: "none", paddingTop: 0 },
-        marginBottom: "xxl",
-      }}
-    >
-      {profileData}
-    </List>
+    <>
+      {usernameStatus !== "" && (
+        <div
```

Comment:
I think we talked about this in a different PR, check with Clare if the banner message is read. In the [Dynamic Banners](https://nypl.github.io/nypl-design-system/reservoir/v3/?path=/docs/components-feedback-banner--docs#dynamic-banners) we say that this wrapper div should always be rendered and have an `aria-live` attribute. If Clare signs off on this, let me know so we can update the docs.